protocol/communities/community.go

```diff
@@ -1877,63 +1884,70 @@ func (o *Community) VerifyGrantSignature(data []byte) (*protobuf.Grant, error) {
 	return grant, nil
 }
 
-func (o *Community) CanPost(pk *ecdsa.PublicKey, chatID string, messageType protobuf.ApplicationMetadataMessage_Type) (bool, error) {
+func (o *Community) CanView(pk *ecdsa.PublicKey, chatID string) bool {
 	if o.config.CommunityDescription.Chats == nil {
-		o.config.Logger.Debug("Community.CanPost: no-chats")
-		return false, nil
+		o.config.Logger.Debug("Community.hasAccessToChat: no-chats")
```

Comment:
Same for the other ones
```suggestion
		o.config.Logger.Debug("Community.CanView: no-chats")
```